Header header

uint16 nl_header
uint32 user_status
uint32 algo_status_0
uint32 algo_status_1
float32 heading
float32 roll
float32 pitch
float32 north_speed
float32 east_speed
float32 vertical_speed
float64 latitude
float64 longitude
float32 altitude
uint32 timestamp
float32 heading_err_sd
float32 roll_err_sd
float32 pitch_err_sd
float32 north_speed_err_sd
float32 east_speed_err_sd
float32 vertical_speed_err_sd
float32 latitude_err_sd
float32 longitude_err_sd
float32 altitude_err_sd
